app/scripts/dimApp.config.js

```diff
@@ -82,7 +82,7 @@
         //Track Our Initial Activity of Starting the App
         $rootScope.trackActivity();
 
-        console.log('DIM v3.7.3 - Please report any errors to https://www.reddit.com/r/destinyitemmanager');
+        console.log('DIM v$DIM_VERSION - Please report any errors to https://www.reddit.com/r/destinyitemmanager');
         dimInfoService.show('20160603v370', {
           title: 'DIM v3.7.3 Released',
           view: 'views/changelog-toaster.html?v=v3.7.3',
```

Comment:
could we also capture these:

```
         dimInfoService.show('20160603v370', {
            title: 'DIM v3.7.3 Released',		            title: 'DIM v3.7.3 Released',
            view: 'views/changelog-toaster.html?v=v3.7.3',		            view: 'views/changelog-toaster.html?v=v3.7.3',
```

if we could also replace `20160603v370` and change it to something like `releaseinfo$DIM_VERSION`?